definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "LabelSolPaymentPara-1",
+    "Label": "Amount to pay: **£${solApplicationFeeInPounds}**",
```

Comment:
are you sure that we can add field to prod and then update it without data migration?

Reply:
This field's only utility is to show as a variable rate in the screen.
It's not important for the logic and it doesn't matter if it's empty.